### ISSUE TITLE
Menu items are shown based on frontend permissions

### ISSUE THREAD

**Issue by victorep** [benfreke/oc-menumanager-plugin#71]
In case the user does not have access to a ceartain page, the link is not displayed.

**Comment by benfreke** [comment]
Hi

This is an enhancement request, right? Only show menu items that the logged in user has permission to view?

**Comment by victorep** [comment]
Yep. I'm also willing to help, I'm very familiar with php but less with laravel, october but I think I can have a crack at it if someone is willing to discuss the plan with me briefly.
